rules-tests/DeadCode/Rector/FunctionLike/TooWideReturnTypeRector/Fixture/arrow_function.php.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Rector\Tests\DeadCode\Rector\FunctionLike\RemoveUnusedReturnTypeRector\Fixture;
```

Comment:
namespace seems copy paste from other rule, make match the rule name

Reply:
please don't resolve conversation unless it resolved :)

Reply:
I'm checking these off as I fix them, I'll push shortly